### ISSUE TITLE
Readme unclear on version of can2040, please add

### ISSUE THREAD

**Issue by mi-hol** [Nate711/can2040-test#1]
https://github.com/Nate711/can2040-test#summary mentions issue but leaves out the important information on what specific version of can 2040 this occured.
Could you please add version?

**Comment by mi-hol** [comment]
project seems dead

**Comment by Nate711** [comment]
Yeah now that the chip shortage is over no point in not using a stm32 for motor drivers
